.pre-commit-config.yaml

```
@@ -69,20 +69,20 @@ repos:
         - flake8-simplify==0.18.2
 
 -   repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.950
+    rev: v0.991
     hooks:
     -   id: mypy
-        name: mypy-habitat-lab
-        files: habitat-lab
+        # https://github.com/python/mypy/issues/4008
+        exclude: "setup.py$|docs/"
```

```suggestion
        exclude: "^docs/|setup.py$"
```
if I remember correctly is the proper way to do it (don't want to exclude any file with docs on the path.